models/offer.go

```diff
@@ -38,6 +42,37 @@ const enabledOffers = `
 		AND offers.enabled = true
 `
 
+func buildScope(enabledOffers string, filterAttrs map[string]string) string {
+	subQueries := []string{enabledOffers}
+	for k, v := range filterAttrs {
+		// TODO: Possible SQL injection
```

Comment:
To avoid sql injections we normally use "?" instead of string interpolations. Is it possible to do the same here?